shared_tests/test_cases.yml

```diff
@@ -16,20 +15,23 @@ book-server_deps:
   - linux
   - quarkus
   withDeps: true
-#book_server_source:
-#  description: book-store basic source analysis
-#  application:
-#    name: book-server
-#    repository:
-#      url: https://github.com/ibraginsky/book-server
-#      kind: git
-#  input: https://github.com/ibraginsky/book-server  # TODO: discuss and unify input application definition
-#  filename: book-server
-#  sources:
-#  - java-ee
-#  - springboot
-#  targets:
-#  - cloud-readiness
-#  - linux
-#  - quarkus
-#  withDeps: false
+  referencesDir: book-server_deps
+
+---
+book-server_source:
```

Comment:
Why having `---` in this file? We should have somethig like (simplified)
```
---
book-server_deps:
  ...
book-server_source:
  ...
```

If this is just for pytest load function, please share the code and I can help with the function parsing it.

Reply:
Done